### PR TITLE
Add demo for image-compression policy

### PR DESCRIPTION
**Pull request by loonybear**

### CODE REVIEW

File: public/demos/image-compression.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  <!-- filled dynamically -->
+</details>
+
+<h2 id="feature-allowed-banner">
+  <span id="allowfeature">...</span>
+</h2>
+
+<p>Click the disallow/allow buttons above to see how the page loads with the policy on and off.
+When the policy is on, you will see images with large file size (byte-per-pixel ratio > 1.25)
+will be rendered in inverted color.</p>
```
Comment (ebidel) [inline public/demos/image-compression.html]: will be rendered in inverted color -> render with inverted colors.

File: public/demos/image-compression.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+<!--
+For images with optimized file size:
+1. image-compression on: images will be rendered accurately.
```
Comment (ebidel) [inline public/demos/image-compression.html]: rendered accurately -> rendered normally.

File: public/js/policies.json [hunk 3]
```diff
@@ -102,4 +102,22 @@
...
+  "url": "/demos/image-compression.html",
+  "what": "Restricts image file size (bits) to be no more than 10 times bigger than its compressed pixels. Images violating this policy will be rendered with inverted color.",
+  "why": "Ensures optimized performance with images by minimizing image file size.",
+  "usage": {
+    "off": "image-compression 'self'",
+    "on": "image-compression 'none'"
+  },
+  "examples": {
+    "header": [
+      "FeaturePolicy: image-compression 'none'"
```
Comment (ebidel) [inline public/js/policies.json]: Feature-Policy

File: public/js/policies.json [hunk 4]
```diff
@@ -102,4 +102,22 @@
       "&lt;iframe allow=\"animations https:\/\/anim8.cdn.com\" src=\"...\"><\/iframe>"
     ]
   }
+}, {
+  "id": "image-compression",
+  "name": "Image Compression",
+  "url": "/demos/image-compression.html",
+  "what": "Restricts image file size (bits) to be no more than 10 times bigger than its compressed pixels. Images violating this policy will be rendered with inverted color.",
```
Comment (ebidel) [inline public/js/policies.json]: Restricts image file size (bits) to be no more than 10 times bigger than its compressed pixels.
-> 
Requires the file size of images (in bits) to be no more than 10x greater than the file size of the image when it is compressed.

Comment (ebidel) [reply public/js/policies.json]: Compressed how much? We should include that.

Comment (loonybear) [reply public/js/policies.json]: no we are comparing it with the pixel area image is rendered

**Comment by loonybear** [comment]
Done. Please take another look. Thanks

**Pull request merged**